.github/workflows/ci.yml

```
@@ -108,12 +108,58 @@ jobs:
         env:
           RUSTFLAGS:
 
-  run-x86_64:
-    name: Run Hermit for Rust (x86_64)
+  run-hermit:
+    name: Run
     runs-on: ubuntu-latest
     defaults:
       run:
         working-directory: kernel
+    strategy:
+      matrix:
+        arch: [x86_64, aarch64]
+        profile: [dev, release]
+        package: [rusty_demo, httpd, testudp, hello_world]
```

```suggestion
        package: [rusty_demo, httpd, testudp, hello_world]
        profile: [dev, release]
```

Looking at this again, it might also make sense to transpose the matrices. What do you think?

---

You are right. That would bring the order in which the tests appear more in line with the hand-written cases from earlier. It may also make sense to have the netdev variable above the profile variable, so that the tree goes `arch → package → netdev → profile`. Which one would you prefer?

---

Yeah, I think that makes sense. 👍